tests/test_core.py

```diff
@@ -25,31 +25,42 @@ class test_PairMatrix(unittest.TestCase):
     def test_init(self):
         """Test construction with different list types."""
 
-        pair_types = ('A','B')
-        pair_dict = {('A','B'):{}, ('B','B'):{}, ('A','A'):{}}
+        types_real = ('A','B')
+        pairs_real  = (('A','B'), ('B','B'), ('A','A'))
```

Comment:
For clarity, let's just call these "types" and "pairs", since "real" doesn't mean much.